docs/DeveloperGuide.md

```diff
@@ -79,7 +79,7 @@ command `delete 3`.
 
 <img src="images\SDforDeletePerson.png" width="800">
```

Comment:
You should have updated more stuffs for your developer  guide. For example the sequence diagram.

you can explain interesting design decisions you made.